doc/10.-Proxy-settings.md

```diff
@@ -0,0 +1,46 @@
+# Proxy settings
+
+The proxy manager allows AD administrators to enforce proxy settings on the clients. Currently, only system-wide proxy settings are supported.
```

Comment:
Maybe we should use `apply | configure` rather than `enforce`? We don't do any checks after applying the proxy, IIRC, so technically we don't really enforce it.